File: updatesproducer/updateapi/update.py
from datetime import datetime
from typing import List

from updatesproducer.updateapi.imedia import IMedia


class Update:
    content: str
    author_id: str
    creation_date: datetime
    url: str
    media: List[IMedia]
    repost: bool
    should_redownload_video: bool

    def __init__(
            self,
            content: str,
            author_id: str,
            creation_date: datetime,
            url: str,
            media: list,
            repost: bool,
            should_redownload_video: bool):
        self.content = content
        self.author_id = author_id
        self.creation_date = creation_date
        self.url = url
        self.media = media
        self.repost = repost
        self.should_redownload_video = should_redownload_video

File: TelegramConsumer/Entities/Update.cs
using System;
using System.Text.Json.Serialization;

namespace TelegramConsumer
{
    public class Update
    {
        [JsonPropertyName("content")]
        public string Content { get; set; }

        [JsonPropertyName("author_id")]
        public string AuthorId { get; set; }

        [JsonPropertyName("creation_date")]
        public DateTime CreationDate { get; set; }

        [JsonPropertyName("url")]
        public string Url { get; set; }

        [JsonPropertyName("media")] 
        public IMedia[] Media { get; set; } = new IMedia[0];

        [JsonPropertyName("repost")]
        public bool Repost { get; set; }

        public override string ToString()
        {
            return $"Url: {Url} - Creation date: {CreationDate} - Media length: {Media.Length} - Repost: {Repost}";
        }
    }
}